chrome/content/zotero/itemTree.jsx

```diff
@@ -1104,6 +1126,16 @@ var ItemTree = class ItemTree extends LibraryTree {
 	}
 	
 	async changeCollectionTreeRow(collectionTreeRow) {
+		// When used outside the Zotero Pane, "collectionTreeRow" is not an actual
+		// tree row being passed in, but rather a simple object that defines necessary properties.
+		// So we need to supplement other CollectionTreeRow properties so that itemTree code
+		// does not complain.
+		// Obviously this is not ideal and would be best refactored so that collection
+		// tree row dependencies are specified separately
+		// and there's a separate method to refresh the items.
+		if (collectionTreeRow.constructor.name === Object.name) {
```

Comment:
Object.name? Do you mean that you're just checking for "Object"? If so you can just use the string.